app/helpers/application_helper.rb

```diff
@@ -1,13 +1,13 @@
 module ApplicationHelper
-  def show_errors_on_base(object)
-    if object.errors.any? && object.errors.messages[:base].present?
-      content_tag(:div, object.errors.full_messages_for(:base).join(", "), class: "alert alert-alert")
+  def show_errors_on_base(obj)
```

Comment:
I'm curious -- why the shortening of the variable name?

Reply:
I was trying to see if there was logic shared by `show_errors_on_base` and `show_errors_inline` - it was easier to compare when the lines are shorter. Do you think its better to spell out `object`?

Reply:
that's cool. I think that code is read more than it is written, so I prefer variable names are verbose enough to reflect what the underlying object is. In cases like this, `object` is fine. I would've also accepted `resource` or `thing`.